RevenueCatUI/Modifiers/ViewExtensions.swift

```diff
@@ -84,11 +84,20 @@ extension View {
         #endif
     }
 
+    private static var isIOSVersionWithCrash: Bool {
```

Comment:
This is going to get called a million times as the view is re-evaluated. I'd cache it 😇

Reply:
great call